test/parallel/test-http-server-write-after-end.js

```diff
@@ -2,15 +2,17 @@
 
 const common = require('../common');
 const http = require('http');
-const assert = require('assert');
 
 // Fix for https://github.com/nodejs/node/issues/14368
 
 const server = http.createServer(handle);
 
 function handle(req, res) {
   res.on('error', common.mustCall((err) => {
-    assert.strictEqual(err.message, 'write after end');
+    common.expectsError({
+      code: 'ERR_STREAM_WRITE_AFTER_END',
+      type: Error
+    })(err);
     server.close();
```

Comment:
Never mind. I thought I could remove the `common.mustCall()` on line `11`, and simplify it to just `common.expectsError()`.

That's what I did in `test-pipe-outgoing-message-data-emitted-after-ended.js` on lines `19-21`.

However, I can't. This block performs `server.close()` in addition to `common.expectsError()`. That additional functionality prevents me from simplifying `common.mustCall() => common.expectsError()`.

Reply:
The way to handle that would be to register two on('error') handlers, one with the `expectsError()` check and the second with the `server.close()` call.